fili-core/src/test/groovy/com/yahoo/bard/webservice/data/QueryBuildingTestingResources.groovy

```diff
@@ -104,6 +108,8 @@ public class QueryBuildingTestingResources {
         DateTimeZone.setDefault(UTC)
         def ages = ["1": "0-10", "2": "11-14", "3": "14-29", "4": "30-40", "5": "41-59", "6": "60+"]
 
+        metadataService = new TestDataSourceMetadataService(Collections.emptyMap())
```

Comment:
In Groovy, it's more idiomatic to use `[:]` rather than `Collections.emptyMap()`